### PR TITLE
BZ1961559: Add nmcli modify command

### PR DESCRIPTION
**Pull request by xenolinux**
[Preview](https://deploy-preview-40088--osdocs.netlify.app/openshift-enterprise/latest/installing/installing_bare_metal_ipi/ipi-install-installation-workflow.html#preparing-the-provisioner-node-for-openshift-install_ipi-install-installation-workflow)
[Fixes](https://bugzilla.redhat.com/show_bug.cgi?id=1961559)
QA contact @achuzhoy 
OCP version: 4.6+

### CODE REVIEW

File: modules/ipi-install-preparing-the-provisioner-node-for-openshift-install.adoc [hunk 1]
```diff
@@ -149,6 +149,13 @@ The IPv6 address can be any address as long as it is not routable via the `barem
 Ensure that UEFI is enabled and UEFI PXE settings are set to the IPv6 protocol when using IPv6 addressing.
 ====
 
+. Modify the provisioner network.
```
Comment (achuzhoy) [inline modules/ipi-install-preparing-the-provisioner-node-for-openshift-install.adoc:152]: This command sets manual IPv4 address on provisioning.
Not clear here if this is always needed. In the previous it shows setting ipv6.

Comment (xenolinux) [reply modules/ipi-install-preparing-the-provisioner-node-for-openshift-install.adoc:152]: So would this change not be needed?

File: modules/ipi-install-preparing-the-provisioner-node-for-openshift-install.adoc [hunk 2]
```diff
@@ -149,6 +149,13 @@ The IPv6 address can be any address as long as it is not routable via the `barem
 Ensure that UEFI is enabled and UEFI PXE settings are set to the IPv6 protocol when using IPv6 addressing.
 ====
 
+. Configure IPv4 address on provisioning connection.
```
Comment (bobfuru) [inline modules/ipi-install-preparing-the-provisioner-node-for-openshift-install.adoc:152]: "provisioning connection" is confusing to me. If the meaning holds, I suggest this:
```suggestion
. Configure the IPv4 address on the `provisioning` network connection.
```

**Review by bobfuru — COMMENTED** [review]
Couple of minor nits. Also, I suspect this should also be applied to 4.6. The BZ was reported against the 4.7 doc set, but the same steps are also in 4.6. Please confirm, thanks!

File: modules/ipi-install-preparing-the-provisioner-node-for-openshift-install.adoc (hunk 2 above)
Comment (xenolinux) [reply modules/ipi-install-preparing-the-provisioner-node-for-openshift-install.adoc:152]: acked

**Comment by xenolinux** [comment]
> Couple of minor nits. Also, I suspect this should also be applied to 4.6. The BZ was reported against the 4.7 doc set, but the same steps are also in 4.6. Please confirm, thanks!

Yes. This PR is valid for 4.6 too.

**Comment by bobfuru** [comment]
LGTM

**Comment by achuzhoy** [comment]
Step 11 is called "Configure networking" and includes this line for setting an ipv6  address on provisioning:
```    nmcli connection modify provisioning ipv6.addresses fd00:1101::1/64 ipv6.method manual```
Because of the above, just setting the ipv4 address in its own step IMHO seems like too much. :)

**Pull request merged**